view/Requirements.php

```diff
@@ -1328,6 +1345,42 @@ public function themedCSS($name, $module = null, $media = null) {
 			$this->css($theme . $css, $media);
 		} elseif($module) {
 			$this->css($module . $css, $media);
+		} elseif (Director::isDev()) {
```

Comment:
If we're throwing exceptions, they should always be thrown, not just in some cases